libcxx/include/__ranges/to.h

```diff
@@ -129,7 +130,7 @@ _LIBCPP_NODISCARD_EXT _LIBCPP_HIDE_FROM_ABI constexpr _Container to(_Range&& __r
     // Try the recursive case.
   } else if constexpr (input_range<range_reference_t<_Range>>) {
     return ranges::to<_Container>(
-        __range | views::transform([](auto&& __elem) {
+        ranges::ref_view(__range) | views::transform([](auto&& __elem) {
```

Comment:
We're already in the `ranges` namespace, and `ranges::ref_view` is not a function (or function template).
```suggestion
        ref_view(__range) | views::transform([](auto&& __elem) {
```